dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -55,12 +56,19 @@
       {% do exceptions.raise_compiler_error(empty_unique_key) %}
     {% endif %}
 
+    {% set inc_predicates_not_list -%}
+      Merge strategy must implement incremental_predicates as a list of predicates.
+    {%- endset %}
+    {% if incremental_predicates is not none and not adapter.is_list(incremental_predicates) %}
```

Comment:
did you try this `if incremental_predicates is iterable`? instead of using a pure python function?

Reply:
Yes, but even a string is iterable. I tried `sequence()` as well but that returns true for both map and list.

Reply:
Ah fair point, I don't have a better suggestion :D